_posts/2017-05-30-predicting-artist-career-stage-with-machine-learning-in-apache-spark.markdown

```diff
@@ -0,0 +1,181 @@
+---
+layout: post
+title: "Predicting Artist Career Stage with Machine Learning in Apache Spark"
+date: 2017-05-30
+comments: true
+author: db
+categories: [Big Data, Apache Spark]
+---
+How established is an artist? Our answer is a fairly subjective value that we call _career stage_. An artist that has passed away and has recently had a retrospective in a major museum correlated with high auction prices has a high career stage value, while a fairly unknown up-and-coming young artist has a low one. The [Art Genome](https://www.artsy.net/categories) historians classified around 25,000 artists in our database by hand, and we predict the career stages of the other 50,000 or so artists using machine learning in Apache Spark.
+
+<!-- more -->
+
+### Extracting Features
+
+First, we retrieve artist features, such as the top price fetched at auction and artist age along with manually assigned career stage value from HDFS using Hive. Here's a subset of the query.
```

Comment:
Neither HDFS or Hive are explained here - might be worth clarification:

"career stage value. We do this by querying Hive, which is the access layer for data stored on our distributed file system (HDFS):"